src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -19,6 +19,7 @@
 use Symfony\Component\DependencyInjection\Exception\RuntimeException;
 use Symfony\Component\DependencyInjection\Reference;
 use Symfony\Component\Messenger\Handler\ChainHandler;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
```

Comment:
Are there two different but similar interfaces or is this a typo? `HandlerInterface` and `MessageHandlerInterface`

Reply:
That's definitely a typo while renaming `HandlerInterface` to `MessageHandlerInterface`. Updated 👍